solr/conf/solrconfig.xml

```diff
@@ -85,7 +85,7 @@
        <str name="echoParams">explicit</str>
        <int name="rows">10</int>
        <str name="q.alt">*:*</str>
-       <str name="mm">2&lt;-1 5&lt;-2 6&lt;90%</str>
+       <str name="mm">5lt;-1 9&lt;90%</str>
```

Comment:
@banukutlu - I looked into the failing tests and it seems like this change is causing the errors.

If you run the following commands on local, you will see the error if you do a search with more than 1 word in the app (e.g. `test 2` causes the error, but `test` does not).

Commands:
```
bundle exec rake solr:create_collection
bundle exec rake solr:update_config
bundle exec rake solr:create_alias
```

I don't know enough about Solr to understand what this change does, but reverting this change and running the commands again gets rid of the error.

Reply:
Update: I think I see the issue. An ampersand is missing from this commit. I'm gonna change it to `<str name="mm">5&lt;-1 9&lt;90%</str>` and I think that will fix it. Stay tuned...

Reply:
yes this makes sense! great catch!